.github/workflows/release_manual.yaml

```diff
@@ -48,13 +48,6 @@ jobs:
         run: |
           make release-all
 
-      - name: "Commit the new changes"
-        run: |
-          export KEY=$(date +%y%m%d)
-          git add .
-          git commit -m "NON-issue: bump version on $KEY"
-
-
```

Comment:
What was the issue?

Reply:
there is no files to commit, so it fails.